uint32 exposure_usecs
